src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1283,7 +1283,28 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string) {
     LOG4CXX_ERROR(logger_, "Service URLs are empty! NOT sending PT to mobile!");
     return;
   }
-  SendMessageToSDK(pt_string, urls.front().url.front());
+
+  static size_t current_app = 0;
+  static size_t current_url = 0;
+  if (current_url >= urls.at(current_app).url.size()) {
+    bool is_default, is_registered, has_urls;
```

Comment:
@tkireva 
- Move variables to the place before their first usage. 
- Please make them `const`
- Use assignment during declaration.